js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -961,6 +1063,25 @@ var KeyboardNavigationController = core.ViewController.inherit({
         }
     },
 
+    _isEditingNavigationMode: function() {
+        return this._isExcelNavigation() && isDefined(this._beginEditingNavigationKey);
+    },
+
+    _excelNavigationBeginEdit: function(originalEvent) {
+        if(!this._isExcelNavigation() || originalEvent.altKey || originalEvent.ctrlKey || this._editingController.isEditing()) {
+            return false;
+        }
+
+        var key = originalEvent.key || String.fromCharCode(originalEvent.keyCode || originalEvent.which);
+
+        if(key && key.length === 1) {
+            this._beginEditingNavigationKey = key;
```

Comment:
Is the _beginEditingNavigationKey reseted after saving?

Reply:
Sure, we have tests for this case, i.e.:
```suggestion
            QUnit.testInActiveWindow("Enter key for not changed editing cell if excelNavigation and batch edit mode", function(assert)
```